wiki/src/test/java/org/exoplatform/platform/ui/qa/wiki/WikiPublishActivityTestIT.java

```diff
@@ -53,6 +55,7 @@ public void setupBeforeMethod() {
    * <li>Post-Condition:</li>
    */
   @Test
+  @Tag("test")
```

Comment:
I inserted this tag to run just one test case (which is under this tag) with maven and i forgot to remove it